src/Campaigns/Blocks/Campaign/edit.tsx

```diff
@@ -0,0 +1,159 @@
+import {useSelect} from '@wordpress/data';
+import {GiveCampaignOptions} from '@givewp/campaigns/types';
+
+declare const window: {
+    GiveCampaignOptions: GiveCampaignOptions;
+} & Window;
+
+const styles = {
+    shared: {
+        position: 'relative',
+        display: 'flex',
+        flexDirection: 'column',
+        gap: 8,
+        padding: 16,
+        borderRadius: 2,
+        color: '#0e0e0e'
+    },
+    title: {
+        fontWeight: 600
+    },
+    link: {
+        color: '#0e0e0e'
+    },
+    enabled: {
+        background: '#f2f2f2',
+        fontSize: 12,
+        lineHeight: 1.33,
+    },
+    disabled: {
+        background: '#fffaf2',
+        borderLeft: '1px solid #f29718',
+    },
+    close: {
+        position: 'absolute',
+        cursor: 'pointer',
+        right: 16,
+        top: 16,
+    }
+} as CSSProperties;
+
+const CloseIcon = () => (
+    <svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+        <path
+            d="M11.805 5.138a.667.667 0 1 0-.943-.943L8 7.057 5.138 4.195a.667.667 0 1 0-.943.943L7.057 8l-2.862 2.862a.667.667 0 1 0 .943.943L8 8.943l2.862 2.862a.667.667 0 1 0 .943-.943L8.943 8l2.862-2.862z"
+            fill="#0E0E0E" />
+    </svg>
+)
+
+export default function Edit({attributes, setAttributes}: BlockEditProps<CampaignBlockType>) {
+    const blockProps = useBlockProps();
+    const [showNotification, setShowNotification] = useState(window.GiveCampaignOptions.admin.showCampaignInteractionNotice);
+    const {campaign, hasResolved} = useCampaign(attributes.campaignId);
+    const adminBaseUrl = useSelect(
+        // @ts-ignore
+        (select) => select('core').getSite()?.url + '/wp-admin/edit.php?post_type=give_forms&page=give-campaigns',
+        []
+    );
+
+    const Notices = () => {
+        if (!attributes.campaignId) {
+            return null;
+        }
+
+        if (campaign.enableCampaignPage) {
+            if (!showNotification) {
+                return null;
+            }
+
+            return (
+                <p style={{...styles['shared'], ...styles['enabled']}}>
+                    <span
+                        style={styles['close']}
+                        onClick={() => {
+                            fetch(window.GiveCampaignOptions.adminUrl + '/admin-ajax.php?action=givewp_campaign_interaction_notice', {method: 'POST'})
+                                .then(() => setShowNotification(false))
+                        }}>
+                        <CloseIcon />
+                    </span>
+                    <span style={styles['title']}>
+                        {__('Campaign interaction', 'give ')}
+                    </span>
+                    <span>
+                        {__('Users will be redirected to campaign page.', 'give')}
+                    </span>
+                </p>
+            )
+        }
+
+        return (
+            <p style={{...styles['shared'], ...styles['disabled']}}>
```

Comment:
We have a `BlockNotice` component in the [form-builder-library](https://github.com/impress-org/form-builder-library/blob/8823490f1a758a8c37b57c9328c8b52b2d62c61b/src/BlockNotice/index.tsx#L4)

Example:
https://github.com/impress-org/give-constant-contact/blob/4315ce42668f6f98e8e8324babf6167c14f9ae4a/src/FormExtension/FormBuilder/Block/edit/BlockInspectorControls/index.tsx#L6